analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/NuGetMetadataFactory.cs

```diff
@@ -23,21 +23,27 @@
 using System.Collections.Immutable;
 using System.IO;
 using System.Linq;
-using System.Reflection;
+using System.Threading;
```

Comment:
Having the `Package` class was a significant improvement in the codebase that simplified the code a lot. Can you implement the change there?
sonar-security is behind in this.

Reply:
I was so happy to drop it :(